pkg/proc/variables_test.go

```diff
@@ -1722,3 +1722,20 @@ func TestBadUnsafePtr(t *testing.T) {
 		}
 	})
 }
+
+func TestCapturedVariable(t *testing.T) {
+	if !goversion.VersionAfterOrEqual(runtime.Version(), 1, 22) {
+		t.Skip("broken")
```

Comment:
Can you add more context here, like `broken in versions prior to 1.22 due to a compiler bug`. Also, does this affect all versions < 1.22 or was the bug introduced in 1.21 and we didn't catch it? If the latter, we should be more specific in the skip.

Finally, can you add a link to the Github issue in a comment somewhere for more context.

Reply:
Done. I hadn't checked before but it really is a regression in 1.21.